local_code_sniffer/vscode-setup.sh

```diff
@@ -0,0 +1,41 @@
+rootdir=$1
+firstdir=`pwd`
+
+if [[ $rootdir == '' ]]
+then
+    echo ""
+    echo "What is the full path to your project's root directory? >  "
+    read rootdir
+fi
+
+echo -e "vendor\n.vscode\ncomposer.json\ncomposer.lock" >> "$rootdir"/.gitignore
+
+echo ""
+echo "Installing Visual Studio Code extensions ..."
+code --install-extension "felixfbecker.php-debug"
+code --install-extension "felixfbecker.php-intellisense"
+code --install-extension "neilbrayfield.php-docblocker"
+code --install-extension "ikappas.phpcs"
+code --install-extension "persoderlind.vscode-phpcbf"
+code --install-extension "dmitrydorofeev.empty-indent"
```

Comment:
This gives a number of warnings, are there better versions of these extensions?
```
Installing Visual Studio Code extensions ...
Installing extensions...
Extension 'felixfbecker.php-debug' is already installed.
Installing extensions...
Extension 'felixfbecker.php-intellisense' is already installed.
Installing extensions...
Installing extension 'neilbrayfield.php-docblocker' v2.1.0...
(node:81371) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
Extension 'neilbrayfield.php-docblocker' v2.1.0 was successfully installed.
Installing extensions...
Installing extension 'ikappas.phpcs' v1.0.5...
(node:81382) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
Extension 'ikappas.phpcs' v1.0.5 was successfully installed.
Installing extensions...
Installing extension 'persoderlind.vscode-phpcbf' v0.0.8...
(node:81393) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
Extension 'persoderlind.vscode-phpcbf' v0.0.8 was successfully installed.
Installing extensions...
Installing extension 'dmitrydorofeev.empty-indent' v0.2.0...
(node:81404) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
Extension 'dmitrydorofeev.empty-indent' v0.2.0 was successfully installed.
```

Reply:
I have looked at another extension that for both phpcs and phpcbf (called PHP Sniffer & Beautifier valeryanm.vscode-phpsab) which has the same deprecation warnings. I will look into others, as well.

Reply:
@jwfuller This seems like it is coming from an npm dependency used in the extension and others that I have checked.